src/Modules/Visualization/ShowField.cc

```diff
@@ -100,6 +100,21 @@ void ShowFieldModule::execute()
 {
   boost::shared_ptr<SCIRun::Field> field = getRequiredInput(Field);
   boost::optional<boost::shared_ptr<SCIRun::Core::Datatypes::ColorMap>> colorMap = getOptionalInput(ColorMapObject);
+  
+  //set the default colormap scaling to be full range if no rescaling was provided.
+  if (colorMap &&
```

Comment:
I was thinking the same thing. But it does confine the min/max to the colormap. field values are not altered in showfield. I'm not sure how else to get the field min/maxes down to the shader, where it is ultimately needed.